core/templates/dev/head/pages/topic-editor-page/editor-tab/topic-editor-tab.directive.ts

```diff
@@ -58,6 +63,65 @@ angular.module('oppia').directive('topicEditorTab', [
               TopicEditorStateService.getCanonicalStorySummaries();
           };
 
+          $scope.getStaticImageUrl = UrlInterpolationService.getStaticImageUrl;
+
+          $scope.showEditThumbnailModal = function() {
```

Comment:
Disable this when user has no edit access.

Reply:
Done.